diff --git a/binarizewolfjolion.cpp b/binarizewolfjolion.cpp
index e92d124..9164c46 100644
--- a/binarizewolfjolion.cpp
+++ b/binarizewolfjolion.cpp
@@ -377,7 +377,7 @@ int main (int argc, char **argv)
 
 
     // Load the image in grayscale mode
-    Mat input = imread(inputname,CV_LOAD_IMAGE_GRAYSCALE);
+    Mat input = imread(inputname, cv::IMREAD_GRAYSCALE);
 
 
     if ((input.rows<=0) || (input.cols<=0)) {
